### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jasonblais**
Just to clarify the `Update` section refers to updating Mattermost Docker to the latest version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,40 +76,44 @@ Put your SSL certificate as `./volumes/web/cert/cert.pem` and the private key th
...
 ```
 
-### Stop
+#### Stop
 ```
 docker-compose stop
 ```
 
-### Update
+### Removing D
```
Comment (pichouk) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -76,40 +76,44 @@ Put your SSL certificate as `./volumes/web/cert/cert.pem` and the private key th
...
-
-### Remove the containers
+#### Remove the data and settings of your Mattermost instance
 ```
-docker-compose stop && docker-compose rm
+sudo rm -rf volumes
 ```
 
-### Remove the data and settings of your Mattermost instance
+## Update Docker to Latest Version
```
Comment (pichouk) [inline README.md]: I suggest to rephrase to "Update Mattermost to latest version" as this section explain how to upgrade Mattermost and not Docker.

File: README.md [hunk 3]
```diff
@@ -76,40 +76,44 @@ Put your SSL certificate as `./volumes/web/cert/cert.pem` and the private key th
...
+
+First, back up your database and your `./data` file [as described here](https://docs.mattermost.com/administration/backup.html#backup).
```
Comment (pichouk) [inline README.md]: This is not the correct procedure to backup your data when using Docker, you need to backup your mounted volumes. I propose the following, feel free to reword 😉 


First, shutdown your containers to back up your data.
```
docker-compose down
```
Back up your mounted volumes to save your data. If you use the default `docker-compose.yml` file proposed on this repository, your data is on `./volumes/` folder.

Then run the following commands.
```
git pull
docker-compose build
docker-compose up -d
```

**Comment by jasonblais** [comment]
Thanks @pichouk! Updates made based on feedback, let me know what you think

**Comment by pichouk** [comment]
LGTM thanks ! 😄

**Pull request merged**